### PR TITLE
Added support for haptic and audio feedback

### PR DESCRIPTION
**Pull request by kober32**
Added `LimeAuthActionFeedback` class that provides simple API for haptic and audio feedback

Implemented haptic/audio feedback in following scenarios:
 - PIN keyboard key pressed
 - successful/failed PIN authorization
 - successful/failed change of pin
 - QR code scanner during an activation process
 - Successfully finished activation

Bumped min sdk version of library to 10.0

This integration can be tried in ios mtoken branch `issues/160-feedback`

### CODE REVIEW

**Comment by petrdvorak** [comment]
Hello, in case we use some sound or images by the 3rd party, we need to make sure we have the commercial license that allows re-distribution...

**Comment by kober32** [comment]
Ad 1) Separated the sound files into own podspec and updated the code to reflect this change
Ad 2) I used different files under Creative Commons license.

**Comment by petrdvorak** [comment]
@kober32 Hello, there are still some changes requested by @hvge. Could you please have a look?

File: LimeAuth.podspec [hunk 1]
```diff
@@ -25,6 +25,7 @@ Pod::Spec.new do |s|
   # 'UI' subspec
   s.subspec 'UI' do |sub|
     sub.source_files = 'Source/UI/**/*.swift'
+    sub.resources = [ 'Source/UI/Sounds/*.m4a' ]
```
[the remark this answers is not in the document]
Comment (kober32) [reply LimeAuth.podspec]: Yes, fixed

File: Source/UI/Common/ActionFeedback/LimeAuthActionFeedback.swift [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+import Foundation
+import AudioToolbox
+import AVFoundation
+
+/// Class that plays haptic and audio feedback for easier use.
+/// For simplicity and configuration of default LimeAuth UI components use `shared` singleton.
+public class LimeAuthActionFeedback {
+    
+    /// Singleton instance used in default UI components
+    public static var shared = LimeAuthActionFeedback()
```
[the remark this answers is not in the document]
Comment (kober32) [reply Source/UI/Common/ActionFeedback/LimeAuthActionFeedback.swift]: Right. Added `actionFeedback` property to both providers

**Comment by kober32** [comment]
Ready to re-review 😅

**Pull request merged**